js/elements.js

```diff
@@ -1,72 +1,76 @@
+import { similarHotels, similarAvatars } from './data.js';
+
+const mapCanvas = document.querySelector('.map__canvas');
 const cardTemplate = document.querySelector('#card').content.querySelector('.popup');
-const card = cardTemplate.cloneNode(true);
 
-card.querySelector('.popup__title').textContent = offer.title;
-card.querySelector('.popup__text--address').textContent = offer.address;
-card.querySelector('.popup__text--price').textContent = `${offer.price} ₽/ночь`;
+const similarOffers = similarHotels();
+const avatars = similarAvatars();
```

Comment:
В этом месте мы и генерируем наши массивы с помощью импортированных функций
```suggestion
const OFFERS_COUNT = 10;

const offers = Array.from({ length: OFFERS_COUNT }, generateObject);
```